distributor-widget/reference.md

```diff
@@ -60,12 +60,28 @@ Sets currency of the Distributor’s localization. Currency code should be in [I
 
 Sets start date for a new availability query, currently loaded availability list is not affected. If a`date`is not a valid Date object or its value isn’t allowed as start date - nothing happens.
 
+* Example - good:
+
+        distributor.setStartDate(new Date(2019, 12, 18))
+
+* Example - wrong:
+
+        distributor.setStartDate(2019-12-18)
+
```

Comment:
@mjesiolowski The examples should be contained withing code blocks, also the wrong is missing commas and it's actually a syntax error.